_data/engine-cli/docker_system_prune.yaml

```diff
@@ -9,10 +10,38 @@ options:
   shorthand: a
   default_value: "false"
   description: Remove all unused images not just dangling ones
-- option: filter
-  description: Provide filter values (e.g. 'until=<timestamp>')
 - option: force
   shorthand: f
   default_value: "false"
   description: Do not prompt for confirmation
+examples: "```bash\n$ docker system prune -a\n\nWARNING! This will remove:\n\t- all
```

Comment:
I think something is really wrong with this example. Everything runs together really weird.